### PR TITLE
seexpr 3.0.1

### PR DESCRIPTION
**Pull request by SeekingMeaning**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SeekingMeaning** [comment]
Possible solution: `-DUSE_PYTHON=FALSE`

**Comment by SeekingMeaning** [comment]
Todo: squash commits

File: Formula/seexpr.rb [hunk 1]
```diff
@@ -18,13 +18,44 @@ class Seexpr < Formula
...
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+                                    |                             
+    EOS
```
Comment (chenrui333) [inline Formula/seexpr.rb:59]: This seems real real.

**Review by chenrui333 — APPROVED** [review]
💯

**Pull request closed without merging**